stable/aws-for-fluent-bit/README.md

```diff
@@ -32,6 +32,8 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `image.repository` | Image to deploy | `amazon/aws-for-fluent-bit` | ✔
 | `image.tag` | Image tag to deploy | `2.21.5`
 | `image.pullPolicy` | Pull policy for the image | `IfNotPresent` | ✔
+| `podSecurityContext` | Security Context for pod | `{}` | 
+| `containerSecurityContext` | Security Context for container | `{}` | 
```

Comment:
nit: security context for Fluent Bit?

Reply:
I meant the text. The description is "Security Context for container" and we could be more descriptive